front/app/containers/Admin/projects/project/analysis/Tasks/index.tsx

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
 import useAnalysisBackgroundTasks from 'api/analysis_background_tasks/useAnalysisBackgroundTasks';
 import { useParams } from 'react-router-dom';
-import { Accordion, Box } from '@citizenlab/cl2-component-library';
+import { Accordion, Box, colors } from '@citizenlab/cl2-component-library';
+import ProgressBar from 'components/UI/ProgressBar';
+import styled from 'styled-components';
+
+const StyledProgressBar = styled(ProgressBar)`
+  height: 8px;
+  width: 100%;
+`;
```

Comment:
Would Iva approve? 😛

Reply:
I'll know next week :)

UI is preliminary in any case, I'll ask her to review (and build further upon) everything in any case.